btree/btree.c

```diff
@@ -7855,21 +7946,36 @@ static void btree_ut_kv_oper_thread_handler(struct btree_ut_thread_info *ti)
 		ut_cb.c_datum = &data;
 
 		while (key_first <= key_last) {
+			/**
+			 * for variable key/value size, the size will increment
+			 * in multiple of 8 after each iteration. The size will
+			 * roundup on reaching MAX_KEY_SIZE. To make sure
+			 * there is atleast one key and size, arr_count is
+			 * incremented by 2.
+			 */
+			arr_count = (key_first % KV_ARR_SIZE) + 2 ;
```

Comment:
extra space between 2 and ;

Reply:
Done.

Reply:
there are similar changes at line no. 8007, 8041, 8044, etc. can you check that too

Reply:
Thanks. Made changes at other places as well.